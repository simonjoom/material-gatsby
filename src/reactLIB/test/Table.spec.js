import React from 'react';
import { shallow } from 'enzyme';
import Table from '../src/Table';

describe('<Table />', () => {
  const wrapper = shallow(
    <Table bordered centered hoverable responsive striped>
      <tr>
        <td>foo</td>
        <td>bar</td>
      </tr>
    </Table>
  );

  test('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
