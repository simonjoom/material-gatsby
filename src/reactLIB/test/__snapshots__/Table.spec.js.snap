// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> renders 1`] = `
<table
  className="centered highlight responsive-table striped bordered"
>
  <tr>
    <td>
      foo
    </td>
    <td>
      bar
    </td>
  </tr>
</table>
`;
